server/config.ts

```diff
@@ -84,6 +84,12 @@ export default {
       agent: new AgentConfig(Number(get('TOKEN_VERIFICATION_API_TIMEOUT_RESPONSE', 5000))),
       enabled: get('TOKEN_VERIFICATION_ENABLED', 'false') === 'true',
     },
+    audit: {
+      queueUrl: get('AUDIT_SQS_QUEUE_URL', 'http://localhost:4566/000000000000/mainQueue', requiredInProduction),
+      region: get('AUDIT_SQS_REGION', 'eu-west-2', requiredInProduction),
+      serviceName: get('AUDIT_SERVICE_NAME', 'UNASSIGNED', requiredInProduction),
+      enabled: get('AUDIT_ENABLED', 'false') === 'true',
+    },
```

Comment:
I'm not sure this should sit under apis. I noticed this was where the prisoner profile project was defining the config though so I put it here for now.

Reply:
I think it might be better to move it out - want to look at automatically registering health checks for apis automatically and this wouldn't fall into that pattern.